tfjs-inference/src/file_handler_test.ts

```diff
@@ -200,12 +196,11 @@ describe('File Handler', () => {
         const handler = new FileHandler(modelJSONPath);
         await handler.load();
 
-        done.fail(
+        fail(
```

Comment:
Where is this `fail` function defined?

Reply:
[`fail` is defined by jasmine](https://jasmine.github.io/api/4.5/global.html#fail).